packages/flutter_tools/lib/src/reporting/custom_dimensions.dart

```diff
@@ -407,3 +407,27 @@ enum CustomDimensionsEnum {
 
   String get cdKey => 'cd${index + 1}';
 }
+
+bool _mapsEqual(Map<dynamic, dynamic>? a, Map<dynamic, dynamic>? b) {
```

Comment:
I'm guessing this is a copy paste from package:usage?

Reply:
Can you substitute the dynamics here for `Object?`?

Reply:
Huh, I don't remember what this is. I think this library isn't reference anymore anway?

Reply:
Ooh, even better, if we can just delete this :)

Reply:
[delete custom_dimensions.dart](https://github.com/flutter/flutter/pull/151518/commits/ca616c0a9e0397b47ad11adcddd76dbe48f9c10a)

Reply:
nice